qa/kerberos/src/main/resources/test.properties

```diff
@@ -0,0 +1,57 @@
+# default settings for Hadoop test
+
+#test.disable.local.es=true
+# ES
+#es.nodes=localhost
+#es.nodes=http://84f306619beadeb16048ae2d4d652ac5.eu-west-1.aws.found.io:9200/
+# by default this is set dynamically based on the port used by the embedded ES instance
+#es.port=9200
+#es.port=9500
+es.batch.size.bytes=1kb
+#es.nodes.client.only=true
+#es.nodes.wan.only=true
+
+# Minimal Security
+es.net.http.auth.user=elastic-admin
+es.net.http.auth.pass=elastic-password
+
+# put pressure on the bulk API
+es.batch.size.entries=3
+es.batch.write.retry.wait=1s
+
+mapred.max.split.size=134217728
+#mapreduce.local.map.tasks.maximum=10
+
+# M&R
+fs.default.name=file:///
+mapred.job.tracker=local
```

Comment:
Do these properties cause the job runner to execute in local mode instead of on the test cluster that is set up?

Reply:
Actually good question. I _thought_ that these were the properties that we had been using. I copied them from test/shared/src/main/resources because (I assumed) that something had changed in the classpath over the years, and [loading](https://github.com/elastic/elasticsearch-hadoop/blob/master/test/shared/src/main/java/org/elasticsearch/hadoop/util/TestSettings.java#L41) had stopped finding that properties file on the classpath. It's possible that there was another test.properties that has been deleted? It looks like that class sets those two properties from system properties though.

Reply:
I don't think any of these were actually being used. I've removed them, and confirmed that when I run the tests I see all of the jobs in the resource manager (http://localhost:8088) so I think we're good here.